common/src/main/java/com/linecorp/centraldogma/common/Author.java

```diff
@@ -32,12 +32,12 @@ public class Author {
     /**
      * The system author.
      */
-    public static final Author SYSTEM = new Author("System", "system@localhost.localdomain");
+    public static final Author SYSTEM = new Author("system", "system@localhost.localdomain");
```

Comment:
There's no reason to have different username with the email especially we are using the username when creating an Author from an email:
https://github.com/line/centraldogma/blob/6bcc0a1580c79cf02c8280c2de240284bf4b1a41/common/src/main/java/com/linecorp/centraldogma/internal/Util.java#L240-L243